usefulUtilities/hiFiCalendar/README.md

```diff
@@ -0,0 +1,25 @@
+# High Fidelity Calendar
+This app lets you integrate with the Google Calendar API to query meeting room schedules.
+
+# Setup Instructions
+1. Setup `hiFiCalendar.html` by copying it to the top level of your C:/ drive and opening a powershell window. Type the following command and press enter, `http-server -p 80` (You must have Node.js installed).
+2. In [Google's console API](https://console.developers.google.com/projectselector2/apis/credentials?supportedpurview=project), setup credentials to get an API Key, Client ID, and Client Secret to paste in `hiFiCalendar.html` (WARNING: DO NOT SHARE THESE OR POST THEM IN A PUBLIC PLACE).  Define your permitted access locations to include http://localhost under Domain Verification.
+
+        // Client ID and API key from the Developer Console
+        var CLIENT_ID = '';
+        var CLIENT_SECRET = '';
+        var API_KEY = '';
+
+3. Copy the calendar IDs into the html file from your Google Calendar, they'll look something like this:
+
+        YourGoogleDomain123456789@resource.calendar.google.com
+
+4. In Interface, create a "Text" entity with the same name you've given your calendar. Paste the server script URL into the `serverScript` field in Entity Properties.
```

Comment:
do you also need to add a child entity to that label text entity that'll display the calendar schedule?